core/src/main/resources/defaults/blueprints/cdp-sdx-medium-ha.bp

```diff
@@ -0,0 +1,428 @@
+{
+  "tags": {
+    "shared_services_ready": true
+  },
+  "description": "CDP 1.0 SDX-HA template with Atlas, HMS, Ranger and other services they are dependent on. Services like HDFS, HBASE have HA",
+  "blueprint": {
+    "cdhVersion": "7.0.0",
+    "displayName": "ha-datalake",
+    "hostTemplates": [
+      {
+        "refName": "master",
+        "roleConfigGroupsRefNames": [
+          "hbase-REGIONSERVER-BASE",
+          "hbase-MASTER-BASE",
+          "hdfs-DATANODE-BASE",
+          "hdfs-FAILOVERCONTROLLER-BASE",
+          "hdfs-JOURNALNODE-BASE",
+          "hdfs-NAMENODE-1",
+          "hive-HIVEMETASTORE-BASE",
+          "solr-SOLR_SERVER-BASE",
+          "zookeeper-SERVER-BASE"
+        ]
+      },
+      {
+        "refName": "master2",
+        "roleConfigGroupsRefNames": [
+          "hdfs-BALANCER-BASE",
+          "hbase-REGIONSERVER-BASE",
+          "hbase-MASTER-BASE",
+          "hdfs-DATANODE-BASE",
+          "hdfs-FAILOVERCONTROLLER-BASE",
+          "hdfs-JOURNALNODE-BASE",
+          "hdfs-NAMENODE-2",
+          "zookeeper-SERVER-BASE"
+        ]
+      },
+      {
+        "refName": "alpha",
+        "roleConfigGroupsRefNames": [
+          "ranger-RANGER_ADMIN-BASE",
+          "ranger-RANGER_USERSYNC-BASE",
+          "atlas-ATLAS_SERVER-1",
+          "hdfs-DATANODE-BASE",
+          "knox-KNOX_GATEWAY-BASE",
+          "ranger-RANGER_TAGSYNC-BASE",
+          "kafka-KAFKA_BROKER-BASE",
+          "zookeeper-SERVER-BASE",
+          "hdfs-JOURNALNODE-BASE"
+        ]
+      },
+      {
+        "refName": "alpha2",
+        "roleConfigGroupsRefNames": [
+          "kafka-GATEWAY-BASE",
+          "atlas-ATLAS_SERVER-BASE",
+          "kafka-KAFKA_BROKER-BASE",
+          "zookeeper-SERVER-BASE"
+        ]
+      },
+      {
+        "refName": "beta",
+        "roleConfigGroupsRefNames": [
+          "hdfs-DATANODE-BASE",
+          "zookeeper-SERVER-BASE"
+        ]
+      },
+      {
+        "cardinality": 1,
+        "refName": "idbroker",
+        "roleConfigGroupsRefNames": [
+          "knox-IDBROKER-BASE"
+        ]
+      }
+    ],
+    "services": [
+      {
+        "refName": "atlas",
+        "serviceType": "ATLAS",
+        "roleConfigGroups": [
+          {
+            "refName": "atlas-ATLAS_SERVER-BASE",
+            "roleType": "ATLAS_SERVER",
+            "base": true,
+            "configs": [
+              {
+                "name": "atlas_kafka_bootstrap_servers",
+                "value": "{{{format-join host_groups.alpha2 format='%s:9092' }}}"
```

Comment:
I think Atlas should be pointed to both Kafka Brokers.  Pointing to local Kafka would be OK if we assume that only the entire host can go down, not the Kafka process itself, but I don't think that's true.

Reply:
Made change to address the comment.